pkg/cmd/server/start/network_args.go

```diff
@@ -16,6 +16,13 @@ type NetworkArgs struct {
 	// ServiceNetworkCIDR is the CIDR string representing the network that service IP
 	// addresses will be allocated from
 	ServiceNetworkCIDR string
+	// IngressIPNetworkCIDR controls the range to assign ingress ips
+	// from for services of type LoadBalancer on bare metal. If empty,
+	// ingress ips will not be assigned. It may contain a single CIDR
+	// that will be allocated from. For security reasons, you should
+	// ensure that this range does not overlap with the CIDRs reserved
+	// for external ips, nodes, pods, or services.
+	IngressIPNetworkCIDR string
```

Comment:
My intent was to allow an admin to set 0.0.0.0/32 to avoid having the default be set.  From what I can tell, if IngressIPNetworkCIDR can't be set by arg, BuildSerializeableMasterConfig will always set the default ingress cidr (unless it conflicts with arg-provided service and cluster net).  Am I missing something, or is that what you intend?